orchestra/views.py

```diff
@@ -183,6 +204,128 @@ def submit_task_assignment(request):
         raise BadRequest(e)
 
 
+@json_view
+@login_required
+@user_passes_test(is_project_admin)
+def project_management_information_api(request):
+    project_id = json.loads(request.body.decode())['project_id']
+    df = work_time_df(Project.objects.filter(id=project_id),
+                      human_only=False, complete_tasks_only=False,
+                      in_progress_iterations=True)
+    project_information = get_project_information(project_id)
+    project_information['project']['admin_url'] = urljoin(
+        settings.ORCHESTRA_URL,
+        urlresolvers.reverse(
+            'admin:orchestra_project_change',
+            args=(project_id,)))
+
+    for slug, task in project_information['tasks'].items():
+        task['admin_url'] = urljoin(
+            settings.ORCHESTRA_URL,
+            urlresolvers.reverse(
+                'admin:orchestra_task_change',
+                args=(task['id'],)))
+
+        for assignment in task['assignments']:
+            assignment['admin_url'] = urljoin(
+                settings.ORCHESTRA_URL,
+                urlresolvers.reverse(
+                    'admin:orchestra_taskassignment_change',
+                    args=(assignment['id'],)))
+
+            iterations = df[(df.worker == assignment['worker']) &
+                            (df.task_id == task['id'])]
+            assignment['iterations'] = []
+            for idx, info in iterations.T.items():
+                iteration = info.to_dict()
+                iteration['end_datetime'] = (
+                    iteration['start_datetime'] + iteration['calendar_time'])
+                del iteration['calendar_time']
+                del iteration['work_time']
+                assignment['iterations'].append(iteration)
+    return project_information
+
+
+@json_view
+@login_required
+@user_passes_test(is_project_admin)
+def reassign_assignment_api(request):
+    worker_username = json.loads(request.body.decode())['worker_username']
```

Comment:
The interface requires the user to provide the name of the worker to reassign, so we have to take in the username here.